### DISCUSSION TITLE
Can Some one suggest me a good first issue

### DISCUSSION THREAD

**Discussion by Bharathhh30** [sanjay-kv/Open-source-Practice#779] [Q&A] [answered]
I am a newbie and finding hard to contribute , can any guide me through my first PR or like help me out with the situation i am in of not finding some beginner friendly or good first issue

**Comment by FatimaAnab** [comment]
If you have little idea about Html or css then you can simply just go to the official website of GSSoC and click on project search for html and css and you'll get some beginner friendly projects that you can work on they are pretty simple and mostly related to styling. you just have to click on the project you like and you can check the repository look for the issue on which you can work on and then simply comment I would like to work on this issue please assign it to me. and then the owner will assign it to you via mail

**Comment by Pranali3103** [comment, marked as answer]
Yeah Sure , If you want to work on 
Web Development Domains : 1.  https://github.com/codervivek5/vigyBag/
                                                2.https://github.com/anuragverma108/SwapReads
                                                3. https://github.com/Priyaaa1/StartConnect-Hub

Data Science and Data analytics :: related issue , go on this GitHub source and find tags for gssoc -level1 : https://github.com/Recode-Hive/Stackoverflow-Analysis

Machine Learning : https://github.com/Niketkumardheeryan/ML-CaPsule

**Comment by MrCodYrohit** [comment]
Try to start with creating contribution.md file, updating readme.md file and learn.md file, as only english you have to write in it.

**Comment by Shelbybosss** [comment]
If you are new to opensource and want to make your first Pull request, go ahead with frontend HTML, CSS or react.js issue, basically issue which are related to frontend.
Also i would suggest that you use Github desktop application for simplicity just clone the repo from there and create a new branch and start working, thereafter push to you branch and open a pull request.

**Comment by aditi75432** [comment]
Try to filter out tech stacks like html , CSS, js etc

**Comment by lovelymahor** [comment]
Visit the GSSOC' 24 repository list and look for projects that interest you. Most repositories have a README file with setup instructions. Understand all the details about project and if you find any issue in the project then you  Look for labels such as good first issue, beginner, or help wanted. These issues are usually simpler and more suitable for newcomers. Make sure you understand what the issue is about. If something is unclear, don't hesitate to ask for clarification in the comments. Ensure that the issue is not already being worked on. Check the comments and see if someone is already assigned. And you can also pull your issue as well.

**Comment by jfmartinz** [comment]
Hi @Bharathhh30, you can contribute on this project: https://github.com/jfmartinz/ResourceHub 

Most of the issues are very easy to do, stacks are basic knoweldge in markdown, html, css and js
